docker/Dockerfile.multi

```
@@ -133,11 +133,17 @@ ENV CCACHE_DIR=/root/.cache/ccache
 ARG GITHUB_MIRROR=""
 ARG BUILD_WHEEL_ARGS="--clean"
 ARG BUILD_WHEEL_SCRIPT="scripts/build_wheel.py"
+ARG PREPARE_WHEEL_FROM_BUILD_STAGE=""
 RUN --mount=type=cache,target=/root/.cache/pip --mount=type=cache,target=${CCACHE_DIR} \
     if [ -n "$GITHUB_MIRROR" ]; then \
       export PIP_INDEX_URL="https://urm.nvidia.com/artifactory/api/pypi/pypi-remote/simple"; \
     fi && \
+    --mount=type=secret,id=GITLAB_TOKEN \
     GITHUB_MIRROR=$GITHUB_MIRROR python3 ${BUILD_WHEEL_SCRIPT} ${BUILD_WHEEL_ARGS}
+    if [ "${PREPARE_WHEEL_FROM_BUILD_STAGE}" != "true" ]; then \
```

Hadolint (2.15.1)

[error] unexpected 'i'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)